index.js

```diff
@@ -1,3 +1,12 @@
+if(typeof window === 'undefined'){
```

Comment:
Really small detail, could you please add a space after `if` and before `{`? Thanks! :smile:

Reply:
Goes for the webpackjsonpreact `if` too